devito/targets/basic.py

```diff
@@ -0,0 +1,153 @@
+from collections import OrderedDict
+from itertools import product
+
+from devito.archinfo import Platform
+from devito.ir.iet import Node
+from devito.logger import warning
+from devito.parameters import configuration
+from devito.targets.common import Graph
+from devito.tools import Singleton
+
+__all__ = ['iet_lower', 'targets', 'targets_registry', 'Target']
```

Comment:
I am not sure whether it is good like targets/Target.possible cause of future confusion...just saying...

Reply:
I don't follow? could you elaborate? you mean the names?

Reply:
Yes, I feel like that so many target*** vaiables may sometime cause confusion.

Reply:
I'm dropping the name "target" in one of the subsequent PRs